bundles/org.eclipse.passage.loc.report.ui/META-INF/MANIFEST.MF

```diff
@@ -13,6 +13,7 @@ Require-Bundle: org.eclipse.passage.loc.yars.api;bundle-version="0.1.0";visibili
  org.eclipse.jface,
  org.eclipse.osgi.services;bundle-version="3.8.0",
  org.eclipse.passage.lic.products;bundle-version="0.5.200",
- org.eclipse.osgi
+ org.eclipse.osgi,
+ org.eclipse.core.runtime
```

Comment:
`org.eclipse.equinox.common` to require less?

Reply:
Have not got the idea, can you please explain a bit wider? `IProgressMonitor` is declared in `core.runtime`. How I can avoid depending on it?

Reply:
It is declared in this package, but the bundle is `org.eclipse.equinox.common`

Reply:
can wait until special work to align our requirements with particular platform release. The idea is to keep LIC as low as possible while use latest improvements for LBC, LOC, LDC.